cfgov/unprocessed/apps/youth-employment-success/js/util.js

```diff
@@ -0,0 +1,56 @@
+let UNDEFINED;
+
+const REDUCER_RETURN_ERROR = 'Reducer must return a state object';
+const processStateFromReducers = reducers => (
+  state = UNDEFINED,
+  action = { type: null }
+) => {
+  /**
+   * If reducers is not an object
+   */
+  const reducersKeys = Object.keys( reducers );
+  const finalState = {};
+  let hasChanged = false;
+
+  for ( let i = 0; i < reducersKeys.length; i++ ) {
+    const key = reducersKeys[i];
+    const reducer = reducers[key];
+
+    /**
+     * if we didnt already get a composite state object, e.g. if this is
```

Comment:
TRIVIAL: Follow proper grammar in code comments (capitalization, ending period, etc.), so it's easier at a glance to see if a comment has been inadvertently truncated, for instance.